src/cases/containers/case-share-complete/case-share-complete.component.html

```diff
@@ -46,12 +47,20 @@ <h2 class="govuk-heading-m case-share-confirm__title">{{ acase.caseTitle }}</h2>
             <span class="hmcts-badge">To be added</span>
           </td>
         </tr>
+        <tr class="govuk-table__row" *ngFor="let user of acase.pendingUnshares;">
+          <td class="govuk-table__cell">{{ user.firstName + ' ' + user.lastName }}</td>
+          <td class="govuk-table__cell">{{ user.email }}</td>
+          <td class="govuk-table__cell">
+            <span class="hmcts-badge hmcts-badge--red">To be removed</span>
+          </td>
+        </tr>
         </tbody>
       </table>
     </div>
   </div>
-  <p class="govuk-body">All your other selected cases were successfully shared.</p>
-  <p class="govuk-body">Try again or <a href="get-help">get help</a> if you are still having problems </p>
+  <p id="add-and-remove-user-info" class="govuk-body" *ngIf="removeUserFromCaseToggleOn$ | async">Access to all your other selected cases was shared or removed.</p>
+  <p id="add-user-info" class="govuk-body" *ngIf="!(removeUserFromCaseToggleOn$ | async)">Access to all your other selected cases was shared.</p>
+  <p class="govuk-body">Try again or <a href="get-help">get help</a> if you’re still having problems</p>
```

Comment:
Appears to be missing a full stop at the end, in terms of consistency with the lines above.

Reply:
The content is from the prototype, but I can add that full stop.